main.cpp

```diff
@@ -95,7 +98,22 @@ Thing 3) sound film production
  */
 
 /*
-Thing 4) pc
+Thing 4) Microphones
+5 properties:
+    1) condeser
+    2) lavalier
+    3) dynamic
+    4) ribbon
+    5) shotgun
+3 things it can do:
```

Comment:
All of these are examples of the same property:  The Microphone's type.  
That is 1 property:  `microphone type`, which can be represented with a string.
come up with 4 more properties.
- [ ] revise